app/src/androidTest/java/ch/epfl/favo/view/MapPageOfflineTest.java

```diff
@@ -0,0 +1,131 @@
+//package ch.epfl.favo.view;
```

Comment:
also what about this file?

Reply:
It's a test class that contains offline tests for the map offline button, which displays a dialog when pressed. Don't know why the dialog is never found by the tests on Cirrus, even when waiting for a long time before the checks. Left it here because I hope someone will make it work in the future and will be able to increase the coverage. I might work on it in the next sprint as well.